torch/quantization/fx/quantize.py

```diff
@@ -620,6 +685,16 @@ def load_arg(a):
                 observed_graph, load_arg)
 
 
+        self.modules = dict(model.named_modules())
```

Comment:
Would it make more sense to pass the top level model around and look up the right module every time?  That way there would be no additional state variables created, so it might be easier to think about the code.  Note: I didn't think through this deeply, more of a question.

Reply:
makes sense, I think we should be able to remove self.modules and replace the access
```
self.modules[node.target]
```
with
```
getattr(model, node.target)
```
will do this in a separate refactor PR

Reply:
just tried this, I think we cannot do it, getattr(model, '') or getattr(model, 'conv.0') is not going to work